test/functional/kernel/nested-loop-reducesum/test-kernel-nested-loop.cpp.in

```diff
@@ -0,0 +1,129 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "RAJA_test-base.hpp"
+#include "RAJA_test-camp.hpp"
+
+#include "RAJA_test-forall-data.hpp"
+#include "RAJA_test-kernel-nested-loop-types.hpp"
+
+#include "RAJA_test-reducepol.hpp"
+
+#include "test-kernel@RESOURCE@nested-loop-@NESTED_LOOP_TYPE@.hpp"
+
+// ----------------------------------------------------------------------------
```

Comment:
Te comments aren't necessary (here and below) because the code is self-explanatory.